### DISCUSSION TITLE
家人被诈骗了 - forecho's Blog

### DISCUSSION THREAD

**Discussion by giscus** [forecho/hugo-blog#47] [Announcements]
# 家人被诈骗了 - forecho's Blog

引言
一个非常让人心痛的事情发生了，家人被诈骗了，从来没觉得诈骗离自己这么近。
这是一个非常痛苦的经历，我希望通过这篇文章来分享一下这个经历，希望能够帮助到更多的人免于受骗。
事情的经过
成为诈骗对象
他在富途上评论了某个股票，然后被人加了好友，然后私聊。刚开始就是闲聊，后面以发送一些电子书为由，让他加了 telegram。
刚开始也是闲聊，他也很少回复，但是只要是有回复，其实就有希望上钩，对方打感情牌，诈骗犯给自己设立一个很有好的形象，人在美国，有钱有闲，炒股炒币经验丰富。

https://blog.forecho.com/the-familys-been-scammed.html

**Comment by HowieHz** [comment]
惨，只能说交学费了，大额的一生只能被骗一次

**Reply by Pathsis** [reply]
也必须被骗一次

**Comment by wookaooo** [comment]
谢谢提醒 之前富途我也加了个人 小美女白富美形象 每天嘘寒问暖 我理智的觉得我这样的丑b 应该不会有这种级别的女的倒贴 就一直冷处理 后来就没找我了😄
